Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -936,6 +936,28 @@ def get_s1_threats_information(self, threat_ids: str) -> dict:
         response = self._http_request(method="GET", url_suffix=f"threats?ids={threat_ids}")
         return response.get("data", [])
 
+    def get_power_query_request(self, account_ids: list, site_ids: list, query: str, from_date: str, to_date: str, limit: int):
+        endpoint_url = 'dv/events/pq'
+        payload = {
+            "accountIds": account_ids,
+            "siteIds": site_ids,
+            "limit": limit,
+            "query": query,
+            "toDate": to_date,
+            "fromDate": from_date
+        }
+        payload = {key: value for key, value in payload.items() if value}
```

Comment:
You can use assign_params found in CommonServerPython.
```suggestion
        payload = assign_params(
            accountIds= account_ids,
            siteIds= site_ids,
            limit=limit,
            query=query,
            toDate=to_date,
            fromDate=from_date
        )
```